lib/tags/profile-settings.tag

```diff
@@ -305,13 +305,15 @@
                                 self.profile.username = username;
                                 self.refs.username.classList.remove('graphjs-error');
                                 self.refs.username.classList.add('graphjs-success');
+                                self.refs.username.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
                                 self.failMessages.includes(failMessage) && self.failMessages.splice(self.failMessages.indexOf(failMessage), 1);
                                 self.successMessages.includes(successMessage) || self.successMessages.push(successMessage);
                                 self.refs.submitProfile.classList.remove('graphjs-loading');
                                 self.update();
                                 self.parent.tags.hasOwnProperty('graphjs-profile-header') && self.parent.tags['graphjs-profile-header'].updateInformation();
                             } else {
                                 self.refs.username.classList.remove('graphjs-success');
+                                self.refs.username.removeAttribute('data-savesuccesstext');
```

Comment:
Please remove this change. self.refs.username refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above